### PR TITLE
Added item number to flex spec.

### PR DESCRIPTION
**Pull request by ajscalise**
Will be used for gather the newton file's ERP reference.

### CODE REVIEW

File: AllfleXML/FlexSpec/FlexSpec.cs [hunk 1]
```diff
@@ -216,6 +216,12 @@ public class Component
         [XmlElement("Color")]
         public string Color { get; set; }
 
+        /// <summary>
+        /// Gets or sets the color.
```
Comment (Oxymoron290) [inline AllfleXML/FlexSpec/FlexSpec.cs]: This property does not get or set the color

Comment (Oxymoron290) [inline AllfleXML/FlexSpec/FlexSpec.cs]: This property does not get or set the color

File: AllfleXML/FlexSpec/FlexSpec.xsd [hunk 2]
```diff
@@ -108,6 +108,14 @@
                               </xs:documentation>
                             </xs:annotation>
                           </xs:element>
+                          <xs:element name="ItemNumber" minOccurs="0" maxOccurs="1">
+                            <xs:annotation>
+                              <xs:appinfo>Component Item Number</xs:appinfo>
+                              <xs:documentation xml:lang="en">
+                                Optional: Reference to the ERP item number that matches the component name and color configuration.
```
Comment (Oxymoron290) [inline AllfleXML/FlexSpec/FlexSpec.xsd]: Change to "Optional: Reference to the ERP item number for the raw material component used in the kit.

**Review by Oxymoron290 — CHANGES_REQUESTED** [review]

**Review by ajscalise — COMMENTED** [review]
Updated

**Pull request merged**